src/rupali/assignment23/Primeno.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Assignment - 23 : 18th April'2021
+
+Write a program with different method to satisfy following requirement. 
+1) validate whether given number is prime or not.
+2) print all the prime numbers given range.
+   input : 40 to 100
+   output : 
+3) count all the prime numbers given range.
+   input : 2 to 100
+   output : 
+4) sum of all the prime numbers given range.
+   input : 2 to 100
+   output : 
+5) Average of all the prime numbers given range.
+   input : 2 to 100
+   output : 
+
+
+
+ */
+package rupali.assignment23;
+
+public class Primeno {
+		void isprime(int num) {
+			int i;
+			for( i=2;i<num;i++) {
```

Comment:
condition should be like this -->. i<=num/2

Simmi - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1119/files